lib/generators/provider/templates/Rakefile

```diff
@@ -1,8 +1,20 @@
+require 'bundler/setup'
+require 'bundler/gem_tasks'
+
 begin
-  require 'bundler/setup'
+  require 'rspec/core/rake_task'
+
+  APP_RAKEFILE = File.expand_path("../spec/manageiq/Rakefile", __FILE__)
+  load 'rails/tasks/engine.rake'
 rescue LoadError
-  puts 'You must `gem install bundler` and `bundle install` to run rake tasks'
 end
 
-APP_RAKEFILE = File.expand_path("../spec/manageiq/Rakefile", __FILE__)
-load 'rails/tasks/engine.rake'
+namespace :spec do
+  desc "Setup environment for specs"
+  task :setup => 'app:test:providers:<%= provider_name %>:setup'
```

Comment:
Good to have divergent opinions 👍 In case we do staged CI https://github.com/ManageIQ/manageiq/issues/13564 it'll be easier to run all specs for other repos within the repo of the PR.

If we solve this different, I'm all for dropping this. In a perfect world this should not be needed, I agree